### PR TITLE
[doc beta] Update docs for rfc176

### PR DESCRIPTION
**Pull request by Parrryy**
PR for #15723 for ember-application. I couldn't seem to find a replacement for `Ember.View`, assuming that just needs to go away but probably in a different pr.

Couldn't see any link related changes.

### CODE REVIEW

File: packages/ember-application/lib/system/application.js [hunk 1]
```diff
@@ -138,14 +146,16 @@ let librariesRegistered = false;
   append views inside it!
 
   To learn more about the events Ember components use, see
-  [components/handling-events](https://guides.emberjs.com/v2.6.0/components/handling-events/#toc_event-names).
+  [components/handling-events](https://guides.emberjs.com/v2.18.0/components/handling-events/#toc_event-names).
```
Comment (Parrryy) [inline packages/ember-application/lib/system/application.js]: Updated this as I was updating this file anyway. Let me know if it's an issue.

**Comment by ynotdraw** [comment]
Correct - from what I've been told so far, `Ember.View` will go away in another PR, so it's fine to leave these spots as-is.

File: packages/ember-application/lib/system/application.js (hunk 1 above)
Comment (locks) [reply packages/ember-application/lib/system/application.js]: cc @jenweber

Comment (toddjordan) [reply packages/ember-application/lib/system/application.js]: should be `https://guides.emberjs.com/current/components/handling-events/#toc_event-names`

Comment (Parrryy) [reply packages/ember-application/lib/system/application.js]: Added change.

**Comment by toddjordan** [comment]
@Parrryy looks great!  Do you mind squashing your commits and prepending the commit message with [doc beta] so we know to put this into 3.0?

Let me know if you are new to git rebase --interactive and need help.

**Pull request closed without merging**